_includes/nav.html

```diff
@@ -22,7 +22,8 @@
               <ul class="dropdown-menu dropdown-menu-dark" aria-labelledby="header-nav-whatWeDo-dropdown">
                 <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'services') %} active" aria-current="page"{% else %}"{% endif %} href="{{ '/services/' | relative_url }}">Services</a></li>
                 <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'equipment') %} active" aria-current="page"{% else %}"{% endif %}  href="{{ '/equipment/' | relative_url }}">Equipment</a></li>
-                <li><hr class="dropdown-divider"></li>
+                <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'FAQ') %} active" aria-current="page"{% else %}"{% endif %}  href="{{ '/FAQ/' | relative_url }}">FAQ</a></li>
+		<li><hr class="dropdown-divider"></li>
```

Comment:
fix indent

Reply:
Not to be rude but out of ignorance...  where?

Reply:
Line 26. You may have mixed tabs and spaces. ![image](https://user-images.githubusercontent.com/4472083/151711876-275c31e9-4a86-4c80-8102-131f0c54408b.png)